compilerlibs/META

```diff
@@ -0,0 +1,43 @@
+version = "[distributed with Ocaml]"
+description = "compiler-libs support library"
+directory= "+compiler-libs"
+
+package "common" (
+  requires = "compiler-libs"
+  version = "[distributed with Ocaml]"
```

Comment:
Is this finally the opportunity to correct the spelling to `OCaml` ?  Or alternatively, it might make sense to just version it with the compiler revision itself.

Reply:
Yes! Fixed. Thanks.

Reply:
Actually you are right, it may be a good idea to include the compiler version here, but that will need a bit more work, so I wait to gather a bit more feedback before doing that.

Reply:
If including the version is a lot of build system hassle, then leaving it as the current string is just fine I think. No tool I'm aware of uses the findlib package version for any structured comparisons.

Reply:
Please make this change before `5.0` is released.
Please include the actual version number.
Please see `let ocaml_version = "@VERSION@"` in your local ocaml source tree.
Thank you so much for merging this PR.

Reply:
PR doing that should be following in the next couple of days, @olafhering and should be in 5.0.0~alpha1, yes.